apps/telehealth-widget/README.md

```diff
@@ -0,0 +1,323 @@
+1. Under the **Destination phone numbers** section, choose **Add phone number**.
+1. Add your users' phone numbers for testing.
+
+Now users with registered phone numbers will receive the appointment reminder via SMS if they are running late. To test the SMS feature, do the following:
+
+1. Suppose you add a doctor and a patient's phone numbers as destination phone numbers in Amazon Pinpoint.
+1. Create an appointment with a doctor and a patient.
+1. Do not open this appointment.
+1. Ensure that a doctor and a patient receive the appointment reminder via SMS at appointment time.
+
+
+#### Step 3: Moving from the SMS sandbox to production
+
+After testing the SMS feature in the SMS sandbox, you can request production access to send SMS messages to any phone number. To do so, follow the instructions in [Moving from the Amazon Pinpoint SMS sandbox to production](https://docs.aws.amazon.com/pinpoint/latest/userguide/channels-sms-awssupport-sandbox.html).
+
+
+### Deploy
+
+Deploy the **AmazonChimeSdkTelehealthWidgetDemo** stack using the default AWS profile and generate `dist/widget.js`. Deployment should complete within 5-10 minutes. See [FAQ](#faq) to learn more about the deployment command or troubleshoot issues.
+```
+cd ~/telehealth-widget
+npm run deploy
+```
+
+### Running the demo
+
+Embed the CSS and JS for widget from CloudFront in your website.
+  
+Step 1: Embed the CSS for widget from CloudFront.
+```html
+  <head>
+    ...
+    <!-- Embed the CSS for widget from CloudFront -->
+    <link href="https://<Cloudfront-DistributionUrl>/index.css" rel="stylesheet" />
+  </head>
+```
+
+Step 2: Embed the JavaScript for widget from CloudFront.
+```html
+<body>
+  <!-- Embed the JS script for widget from CloudFront -->
+  <script src="https://<Cloudfront-DistributionUrl>/index.js"></script>
+</body>
+```
+
+Step 3: Load the widget in your website.
+```html
+<body>
+  ...
+  <script>
+    window.initAmazonChimeSDKWidget();
+  </script>
+</body>
+```
+
+Now, run your website and you will see the widget embedded in your website.
+
+## Clean up
+
+Deploying the stack may incur charges to your AWS account. Use the following steps to delete resources.
+```
+cd ~/telehealth-widget
+npm run destroy
+```
+
+Following logs will be printed to terminal, please input 'y' when prompted to confirm deletion:
+```
+> chime-sdk-telehealth-widget-demo@1.0.0 destroy
+> cd backend && npm run destroy
+
+
+> chime-sdk-telehealth-widget--backend-demo@1.0.0 destroy
+> cdk destroy
+
+Are you sure you want to delete: AmazonChimeSdkTelehealthWidgetDemo (y/n)? y
+AmazonChimeSdkTelehealthWidgetDemo: destroying...
+ ✅  AmazonChimeSdkTelehealthWidgetDemo: destroyed
+```
+
+If any issue deleting the stack through command line, do the following to delete it from the AWS management console:
+
+1. Open the [AWS CloudFormation console](https://console.aws.amazon.com/cloudformation/).
+2. Delete the *AmazonChimeSdkTelehealthWidgetDemo* stack.
```

Comment:
nit: Bold **AmazonChimeSdkTelehealthWidgetDemo** and *CDKToolkit** for consistency with other resource names.

Reply:
Yes seems like quip copy paste 😅 Thanks for catching.